### PR TITLE
Add post for go-ipfs 0.4.3 release

### PR DESCRIPTION
**Pull request by lgierth**
Rendered preview here: https://ipfs.io/ipfs/QmRmS6SWruG1PsQQvsUchRStejAKPPvVbMA8KuDGqHsmui/

### CODE REVIEW

File: src/19-ipfs-0-4-3-released/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+[Take a look at the Changelog](https://github.com/ipfs/go-ipfs/blob/master/CHANGELOG.md).
+
+## How to upgrade
+
+Depending on how you initially installed IPFS, there are several ways to
+upgrade. If you installed IPFS with a pre-built binary, you can either head over
+to [dist.ipfs.io](https://dist.ipfs.io/#go-ipfs) and grab the latest version
+from there. Or alternatively, from the same page you can grab the `ipfs-update`
+binary, and use it to perform the upgrade for you. If you installed from
+source, you can simply update your git repo to the `v0.4.3` tag, run `make install`.
```
Comment (RichardLitt) [inline src/19-ipfs-0-4-3-released/index.md]: then run

**Comment by RichardLitt** [comment]
Looks mostly good to me! one small comment.

File: src/19-ipfs-0-4-3-released/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- The daemon now automatically runs fs-repo migrations.
+- The daemon got new flags:
+  - `ipfs daemon --offline` can be used to disable all swarm networking.
+  - `ipfs daemon --migrate` automatically runs any pending fs-repo migrations.
+- The Datastore got a few more config options:
+  - `Datastore.HashOnRead` allows you to verify data on read access.
+  - `Datastore.BloomFilterSize` tunes the new lookup cache.
+
+And plenty of bug fixes as well as improvements to documentation and test coverage.
+[Take a look at the Changelog](https://github.com/ipfs/go-ipfs/blob/master/CHANGELOG.md).
```
Comment (jbenet) [inline src/19-ipfs-0-4-3-released/index.md]: maybe we should include the changelog here? that way people get it on their email and they can read it right there

Comment (jbenet) [inline src/19-ipfs-0-4-3-released/index.md]: at the end, after the "how to upgrade"

**Comment by jbenet** [comment]
LGTM

**Comment by lgierth** [comment]
Here we go: https://ipfs.io/blog/

**Pull request merged**